library/c/api.go

```diff
@@ -203,14 +203,6 @@ func waku_content_topic(applicationName *C.char, applicationVersion C.uint, cont
 	return execOkCB(onOkCb, contentTopic.String())
 }
 
-// Create a pubsub topic string according to RFC 23
-//
-//export waku_pubsub_topic
-func waku_pubsub_topic(name *C.char, encoding *C.char, onOkCb C.WakuCallBack) C.int {
```

Comment:
I am wondering if we should have an API to create sharded pubsub topic based on cluster and shard.
WDYT @richard-ramos 
Atleast until all API's are migrated to only take either contentTopic or cluster/shard.

Reply:
Yeah, sounds good.
@harsh-98 this can be done as a separate PR, i think, to get this one merged asap :)